provider/alibabacloud/alibaba_cloud.go

```diff
@@ -373,24 +374,24 @@ func (p *AlibabaCloudProvider) groupRecords(records []alidns.Record) (endpointMa
 func (p *AlibabaCloudProvider) records() ([]alidns.Record, error) {
 	log.Infof("Retrieving Alibaba Cloud DNS Domain Records")
 	var results []alidns.Record
-
+	hostedZoneNames, err := p.getDomainList()
+	if err != nil {
+		log.Errorf("AlibabaCloudProvider getDomainList error %v", err)
```

Comment:
```suggestion
		log.Errorf("getDomainList: %w", err)
```

Reply:
go throws the warning `github.com/sirupsen/logrus.Errorf does not support error-wrapping directive %wprintf`

Reply:
i'll switch to using Errof from fmt package

Reply:
Ah, missed the `log.`. Never mind.